### PR TITLE
Devnet Fix

### PR DESCRIPTION
**Pull request by liam-lai**
1. Fix docker compose file
2. Disable cache on docker file for always pull latest commit but keep basic package installed. Speed up rebuild
3. Every time docker-up will re-create image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Your Masternode Name is test01
 
 **For Devnet**
 ```
-sudo su -c "bash <(wget -qO- https://raw.githubusercontent.com/XinFinOrg/XinFin-Node/master/setup/bootstrap.sh)" root
+sudo su -c "bash <(wget -qO- https://raw.githubusercontent.com/hash-laboratories-au/XinFin-Node/master/setup/bootstrap.sh)" root
```
Comment (wjrjerome) [inline README.md:102]: change it back to XinFinOrg

File: devnet/Dockerfile [hunk 2]
```diff
@@ -2,17 +2,16 @@ FROM golang:1.14
...
-RUN \
-    apt-get update && apt-get install -y git build-essential && \
-    git clone https://github.com/XinFinOrg/XDPoSChain.git xdcchain && \
-	(cd xdcchain && git checkout --detach ea5ca4f1be8889244a2019b00b6e3a9564be0d8c && make)
+RUN apt-get update && apt-get install -y git build-essential
 
-RUN cp /work/xdcchain/build/bin/XDC /usr/bin && chmod +x /usr/bin/XDC && \
-    rm -rf xdcchain
+RUN git clone https://github.com/hash-laboratories-au/XDPoSChain.git && \
+        (cd XDPoSChain && git checkout dev-upgrade && make)
```
Comment (wjrjerome) [inline devnet/Dockerfile:8]: hmmmm.... why change this tho? i think it's best not to modify the existing logic, it's easier for testing.
But i think the proper way of fixing is through proper CI CD with docker images

File: devnet/cleanup.sh [hunk 3]
```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
-rm -rf xdcchain-*
+# rm -rf xdcchain-*
```
Comment (wjrjerome) [inline devnet/cleanup.sh:2]: you still need to clean up the `xdcchain-*` no?

File: README.md (hunk 1 above)
Comment (liam-lai) [reply README.md:102]: after we finish test, then I will change it back

File: devnet/Dockerfile (hunk 2 above)
Comment (liam-lai) [reply devnet/Dockerfile:8]: This is for devnet online test only. Yes, if we have CICD then the process can be different

File: devnet/cleanup.sh (hunk 3 above)
Comment (liam-lai) [reply devnet/cleanup.sh:2]: yes, I think I will make docker clean up and file clean up seperated

**Pull request merged**